src/main/log/logManager.ts

```diff
@@ -45,15 +51,15 @@ export class LogManager implements ExtensionComponent {
     public logError(error: Error, shouldToast: boolean) {
         this.setFailed();
         this.logMessage(error.name, 'ERR');
+        this.logMessage(error.message, 'ERR');
```

Comment:
Now that we have the 2 lines above, does this mean we'll have two log lines for each error? One with the error name and one with the error message? If so, is that what we want to have? Why not have both in one log line?